javascript/README.md

```diff
@@ -0,0 +1,11 @@
+### Example of integrating Salt Edge API with JavaScript.
+
+* After clone this code, in folder with files run any file server (like python -m SimpleHTTPServer) or another.
+
+* In your [Saltedge Profile Settings](https://www.saltedge.com/clients/profile/edit) set this address to home url and save.
+
+* Then open in browser page with ip address (like http://192.168.88.88 if use local server)
+
+* Go to [Profile Data Secrets](https://www.saltedge.com/clients/profile/secrets) and Copy Paste data to JavaScript Application Page.
+
+* Now you can connect your banks and receive data.
```

Comment:
Try to elaborate and provide instructions on how to connect and see example data